worlds/no100f/__init__.py

```diff
@@ -181,8 +183,11 @@ def check_version(cls, opened_zipfile: zipfile.ZipFile) -> bool:
         return True
 
 
+def get_filler_item_name(self) -> str:
+    return self.random.choice("Scooby Snack", "Scooby Snack Box")
```

Comment:
This is supposed to go in the World class. I usually put it after `create_item` for organization purposes